### DISCUSSION TITLE
Can I use `ExceptionGroup` in validators?

### DISCUSSION THREAD

**Discussion by vadim-su** [pydantic/pydantic#7661] [Question] [answered]
Is `ExceptionGroup` supported? I want to validate multiple fields in a validator and return a batch of errors to a user.

**Comment by samuelcolvin** [comment, marked as answer]
We already raise and error with details of each field error.

Formal exception group support is coming soon #7626.

**Reply by Prometheus3375** [reply]
Hello, was the support added? When raising `ExceptionGroup` with a list of `ValueError`, the final exception is `ExceptionGroup`, not Pydantic `ValidationError`.
I am using `exceptiongroup` backport for 3.10 and Pydantic version 2.7.1.
